orchestra/communication/staffing.py

```diff
@@ -197,3 +227,56 @@ def get_available_requests(worker):
             reverse('orchestra:communication:available_staffing_requests'))
         contexts.append(metadata)
     return contexts
+
+
+def get_inquiries_per_worker_count(request):
+    inq_count = (request.inquiries.all()
+                 .values('communication_preference__worker')
+                 .annotate(Count('communication_preference__worker')))
+    return inq_count[0]['communication_preference__worker__count']
+
+
+def check_unstaffed_tasks():
+    # Get all requests without winners
+    task_values = (
+        Task.objects.all()
+        .filter(staffing_requests__inquiries__responses__is_winner=False)
+        .exclude(staffing_requests__isnull=True)
+        .exclude(staffing_requests__inquiries__isnull=True)
+        .order_by('-start_datetime')
+        .values('staffing_requests__required_role_counter', 'id')
+        .annotate(Count('staffing_requests__id')))
+
+    for task_value in task_values:
+        if task_value['staffing_requests__id__count'] > MAX_REQUEST_COUNT:
+            # Send to #staffing channel.
+            message_experts_slack_group(
+                STAFFING_GROUP_ID,
+                ('No winner request for task {}!'
+                 .format(task_value['id'])))
+            continue
+        required_role_counter = task_value[
+            'staffing_requests__required_role_counter']
+        request = (
+            StaffBotRequest.objects.filter(
+                task=task_value['id'],
+                required_role_counter=required_role_counter)
+            .order_by('-created_at'))[0]
+
+        if request.created_at < timezone.now() - timedelta(
```

Comment:
if you delete `MAX_REQUEST_COUNT` logic, you can push this request.created_at into the task_values filter :)

Reply:
all g